pandas/tests/arrays/string_/test_string.py

```diff
@@ -277,3 +277,13 @@ def test_value_counts_na():
     result = arr.value_counts(dropna=True)
     expected = pd.Series([2, 1], index=["a", "b"], dtype="Int64")
     tm.assert_series_equal(result, expected)
+
+
+def test_normalize_value_counts():
```

Comment:
can you simply expand this test:

```pandas/tests/extension/base/methods.py:    def test_value_counts(self, all_data, dropna)```

instead which generically tests all EAs for value_counts